chapters/chap03-sec03.tex

```diff
@@ -114,45 +117,46 @@ \subsection{Instantiating the packing functions}\label{instantiation:statement:p
             \t \t out \gets out +_{\FFx{\rBN}} 2^{\len{x}-1-i} \\
             \pcreturn out;
         }
-        \caption{Packing algorithm in big endian.}\label{zeth-protocol:fig:packing-alg}
+        \caption{Algorithm to pack bits into a field element.}\label{zeth-protocol:fig:packing-alg}
     \end{minipage}%
     \begin{minipage}[t]{.6\textwidth}
         \centering
         \procedure{\packResBits{\nfs{}, \vin, \vout, \hsig, \htags{}}}{%
             out \gets []; r \gets \epsilon; \\
             r \gets \slice{\vin}{\floor{\zvalueLen / \bnFieldBitCap} \cdot \bnFieldBitCap}{}; \\
-            r \gets r \concat \slice{\vout}{\floor{\zvalueLen / \bnFieldBitCap} \cdot \bnFieldBitCap}{}; \\
-            r \gets r \concat \slice{\hsig}{\floor{\crhhsigOutLen / \bnFieldBitCap} \cdot \bnFieldBitCap}{}; \\
+            r \gets \slice{\vout}{\floor{\zvalueLen / \bnFieldBitCap} \cdot \bnFieldBitCap}{} \concat r; \\
```

Comment:
Here again, can't we use the `\beta` notations?

Reply:
Switched these to use `\beta_*`.  (I also removed the slice in`vout` and `vin` since they are always fully packed alongside the residual bits.)